tuteria_application_test/users/models.py

```diff
@@ -5,7 +5,9 @@
 from django.core.urlresolvers import reverse
 from django.db import models
 from django.utils.encoding import python_2_unicode_compatible
-from django.utils.translation import ugettext_lazy as _
```

Comment:
What is the point of doing this? How did it affect your current solution that you had to change? And u ended up not using what u changed.